bin/deepstate/core/fuzz.py

```diff
@@ -716,13 +716,15 @@ def run(self, runner: Optional[str] = None, no_exec: bool = False):
               (self.proc.returncode == 1 and self.name == "libFuzzer"):
             L.info("Fuzzer %s (PID %d) exited with return code %d.",
                       self.name, self.proc.pid, self.proc.returncode)
+            self.fuzzer_return_code = 0
```

Comment:
We may set this below [line 97](https://github.com/trailofbits/deepstate/blob/3a94cdf0bb7d96db1088983a537b1fcbcc1dc518/bin/deepstate/core/fuzz.py#L97). Also please extend __init__ docstring with description what fuzzer_return_code is and that it is returned by main method.

Reply:
OK.  I kept the self.fuzzer_return_code = 0 on 719/721 to keep consistent with the 'else' case even though it should always be 0.